### PR TITLE
[TS-221] Add pprof with set engine and port

### PR DESCRIPTION
**Pull request by pcherednichenko**

### CODE REVIEW

File: examples/cli/main.go [hunk 1]
```diff
@@ -57,6 +57,7 @@ func main() {
 				Destination: &echo,
 			}
 		}),
+		cli.Pprof(nil, ":6061"),
```
Comment (kpeo) [inline examples/cli/main.go]: may be it's better put it into constant or config?

Comment (im-kulikov) [reply examples/cli/main.go]: @kpeo this is just an example..

File: pprof/pprof.go [hunk 2]
```diff
@@ -2,11 +2,43 @@ package pprof
...
+	"os/signal"
 	"strings"
+	"syscall"
 
+	"github.com/cryptopay-dev/yaga/logger"
 	"github.com/cryptopay-dev/yaga/web"
 )
 
+const (
+	defaultPprofPort = ":6060"
```
Comment (kpeo) [inline pprof/pprof.go]: the same. const or config?

File: cli/options.go [hunk 3]
```diff
@@ -17,6 +18,9 @@ type Options struct {
 	Redis           *redis.Client
 	Users           []cli.Author
 	Debug           bool
+	Engine          *web.Engine
```
Comment (im-kulikov) [inline cli/options.go]: for what?

File: pprof/pprof.go (hunk 2 above)
Comment (kpeo) [inline pprof/pprof.go]: same as before. may be better put the port into const or config?

**Review by jenchik — APPROVED** [review]

**Pull request merged**